src/BTree.h

```diff
@@ -550,7 +550,7 @@ class btree {
             assert(!this->parent || this->parent->lock.is_write_locked());
             assert((this->parent != nullptr) || root_lock.is_write_locked());
 #endif
-            assert(this->numElements == maxKeys);
+            assert(this->numElements == maxKeys && true);
```

Comment:
Why the `&& true`?

Reply:
sanity check for that the debugger was looking at the right function in the inheritance chain